content/markdown/02-working-with-data/02-06-factors.rmd

```diff
@@ -0,0 +1,189 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Factors - Releveling & Recoding
+**Author:** Cole Brookson
+**Date:** 20 July 2022
+
+```{r, echo=FALSE}
+options(tidyverse.quiet = TRUE)
+```
+
+
+## Intro to Factors 
+
+Factors are an important but frustrating special data type in R. They can be thought of as "categorical" variables, in the sense that they can only take on a limited number of values. R uses a vector of integer values to store a factor, with a corresponding set of character values that are displayed while using the factor.
+
+They are most commonly used to group numerical variables, and therefore commonly come into play when working with regression-related topics that require either categorical fixed effects or a random effect. An example might be the site at which some data were collected. In a study where there are only six sites, it might be the case that we need to use a *random effect* to control for variation between sites. In that scenario, it would be helpful to treat the variable **Site** as a factor in R. 
```

Comment:
Thinking of the student reader here, it might be confusing to start talking about fixed effects / random effects. Perhaps even just by linking to a section on those topics here might help so students can see briefly what that means if they're interested or confused?